pkg/data/components/OTelGRPCExporter.yaml

```diff
@@ -24,13 +24,22 @@ ports:
     direction: input
     type: OTelLogs
 properties:
+  - name: Headers
+    summary: Headers to emit when sending gRPC traffic.
+    description: |
+      Sending data to a backend may require additional headers to be
+      configured. This property supports sending a map of header keys and
+      values.
+    type: map
+    validations: []
   - name: Host
     summary: The hostname or IP address to send data to
     description: |
       Hostname or IP address on which to send outgoing GRPC traffic.
     type: string
     validations: [nonblank]
     default: https://api.honeycomb.io
+    advanced: true
```

Comment:
what does it mean for a property to be advanced?

Reply:
I updated the PR description above to include that information.